### PR TITLE
📬 Correios track command

### PR DESCRIPTION
**Pull request by Doges**

### CODE REVIEW

**Comment by Doges** [comment]
RFR

File: src/commands/government/correios.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const { Command, CommandError, SwitchbladeEmbed, Constants } = require('../../')
+const moment = require('moment')
+
+module.exports = class Correios extends Command {
+  constructor (client) {
+    super({
+      name: 'correios',
+      alias: [ 'trackCorreios' ],
+      category: 'government',
```
Comment (pedrofracassi) [inline src/commands/government/correios.js:9]: Does this have a string?

**Comment by perronosaurio** [comment]
Why only br?

**Comment by Doges** [comment]
> 
> 
> Why only br?

bc i can't find the 17track public api

File: src/commands/government/correios.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const { Command, CommandError, SwitchbladeEmbed, Constants } = require('../../')
+const moment = require('moment')
```
[the remark this answers is not in the document]
Comment (Doges) [reply src/commands/government/correios.js:2]: > 
> 
> Maybe user another date library? https://github.com/you-dont-need/You-Dont-Need-Momentjs

PS: i'm not using moment to parse the dates, but for the i18n

[the remark this answers is not in the document]
Comment (Doges) [reply src/commands/government/correios.js:2]: PPS: the timestamp already parse the dates, so thats not necessary

[the remark this answers is not in the document]
Comment (fernando457829) [reply src/commands/government/correios.js:2]: Ok, I didn't see it.

**Comment by fernando457829** [comment]
LGTM

**Pull request merged**